README.md: Added description (#12)
Mixss/enauczanie-feed: README.md

# eNauczanie Feed

Modify your eNauczanie dashboard to communicate with others!

> Feed tab shows 3 newest messages sent by others. To send new message select
> `Nowa wiadomość` then type your message then/or paste link to an image. When you
> send it everybody will se it!

Reviewer: ```suggestion
The `Feed` tab shows 3 newest messages sent by other users. To send a new message select
`Nowa wiadomość` then type your message and/or paste link to an image. When you
send it everybody else will see it!
```
